components/compositing/compositor.rs

```diff
@@ -1518,6 +1527,22 @@ impl IOCompositor {
                     },
                     _ => {},
                 }
+
+                if let Some(_lcp) = self.lcp_detector.calculate_largest_contentful_paint(
+                    paint_time,
+                    current_epoch,
+                    pipeline_id.into(),
+                ) {
+                    #[cfg(feature = "tracing")]
+                    let _ = tracing::debug_span!(
+                        "largest-contentful-paint",
+                        servo_profiling = true,
+                        timestamp =
+                            (_lcp.paint_time - CrossProcessInstant::epoch()).whole_microseconds(),
+                    )
+                    .entered();
+                    println!("{_lcp:?}");
```

Comment:
I guess this was left over from debugging? please prefer using `log` over `println`, since log allows filtering etc.